home/views.py

```diff
@@ -7,3 +9,37 @@ def about(request):
 
 def landingpage(request):
     return render(request, 'home/landingpage.html')
+
+
+class QuestionDetailView(DetailView):
+    model = Question
+
+    def get_context_data(self, **kwargs):
```

Comment:
I think this is not a good function design - it has many very different things included in it, think about how difficult it would be to write tests for it.

I think it should be broken apart and most stuff in it moved out to model functions.

Reply:
fixed